<?xml version="1.0"?>
<launch>

	<!-- arg -->
	<arg name="sensor_topic"	default="/peg_sensor_classifier"/>
	<arg name="action_topic" 	default="peg_link"/>
	<arg name="socket_type"		default="one"/>
	<arg name="path_sensor_model"   default="$(find objects)/meshes/plug/config/X.txt"/>

	<arg name="table_urdf" value="'$(find peg_in_hole)/urdf/One/table_sw_one.urdf'" />

     	<node name="peg_pf_node" pkg="peg_filter" type="peg_filter_node"  args="-sensor_topic $(arg sensor_topic) -action_topic $(arg action_topic) -urdf $(arg table_urdf) -path_sensor_model $(arg path_sensor_model)"   launch-prefix="xterm -e gdb --args"  required="true" output="screen"/>


</launch>
